### DISCUSSION TITLE
Multiple modified A in the same kmer

### DISCUSSION THREAD

**Discussion by obenno** [GoekeLab/xpore#102] [Q&A]
Hi,

I would like to ask is there anyway to exclude situations with multiple modified A in the same kmer which could confound signal means? Is the single modified A which localize in the center of the 5 mer the assumption of `xpore`?

Best regards,
obenno

**Comment by ploy-np** [comment]
Hi @obenno, 

The assumption that the single modified A locates in the centre of k-mer was used for the validation purpose only in the paper. In fact, the signal profiles correspond to all 5 bases in a 5-mer. So, in theory, if a modified k-mer, either one base or more, has different signal profiles and both are different from the control, xpore can detect all cases. We can further analyse the `mu_mod` and `sigma_mod` values for those k-mers, which may help distinguish different kinds of modifications i.e. single or multiple modification.

**Comment by obenno** [comment]
Hi @ploy-np,

Thanks for your reply.

Does that mean the "modified" distribution inferred is actually a mix of multiple types of modified distribution, with one "A", two "A" etc. ? It will be great to have your kind suggestions on how to discriminate them from the result. In our result of soybean nanopore data, DRACH sites only account for around 10% of the total NNANN sites, any chance this was caused by multiple modified A in the 5-mer? Thank you in advance!

Best Regard,
obenno
